thunder/benchmarks/targets.py

```diff
@@ -812,3 +815,37 @@ def test_resnet50(benchmark, executor: Callable, compute_type: ComputeType):
     fn = executor(b.fn())
 
     benchmark_for_compute_type(compute_type, benchmark, fn, args, kwargs)
+
+
+#
+# Torchbench benchmarks
+#
+
+torchbench_models = []
+if importlib.util.find_spec("torchbenchmark"):
```

Comment:
Should we add `torchbenchmark` to one of the requirements.txt file (maybe test.txt) so that it would be installed.

Reply:
For this I am not sure it can be done this easy, torchbenchmark requires the extra step of installing the models which is not done by pip. I think the installation should be left to the user following the guide at https://github.com/pytorch/benchmark/tree/main . I can add a comment tho detailing this